File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getBlockByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockHash = '0xb8deae63002d2b6aa33247c8ef545383ee0fd2282ac9b49dbbb74114389ddb5c'
    const transactionObject = true
    provider.eth.getBlockByHash(blockHash, transactionObject, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/eth/block/getBlockByHash.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xb8deae63002d2b6aa33247c8ef545383ee0fd2282ac9b49dbbb74114389ddb5c"
transactionObject = True

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_block_by_hash(blockHash, transactionObject)

print(eth_response)
